ambry-commons/src/main/java/com.github.ambry.commons/ServerErrorCode.java

```diff
@@ -27,5 +27,7 @@
   Partition_Unknown,
   Disk_Unavailable,
   Partition_ReadOnly,
-  Unknown_Error
+  Unknown_Error,
```

Comment:
@vgkholla let us add a comment similar to the one for `RequestResponseType` to make it clear that ordering of enums used in protocol should not be changed.